tests/starship/configs/local.yaml

```diff
@@ -0,0 +1,22 @@
+chains:
+  - name: simapp
+    type: simapp
+    image: ghcr.io/cosmology-tech/starship/simapp:v0.47.3
```

Comment:
all images should be looking at (https://github.com/cosmos/cosmos-sdk/pkgs/container/simapp) that way we are testing the latest from the sdk